addons/website_event/views/event_templates_list.xml

```diff
@@ -325,6 +325,7 @@
                         <div class="d-block h-100 w-100">
                             <t t-call="website.record_cover">
                                 <t t-set="_record" t-value="event"/>
+                                <t t-set="additional_classes" t-value="h-100"/>
```

Comment:
How is this not crashing? It should either be `t-value="'h-100'"` or `t-valuef="h-100"`, no?